content/desktop/get-started.md

```diff
@@ -55,7 +55,7 @@ $ gpg --generate-key
 
 The following is an example similar to what you see once you run the previous command:
 
-```console
+```hcl {hl_lines=12}
```

Comment:
Should keep console here, HCL is hashicorp configuration language. Line hl is defined using the hl_lines option and has nothing to do w/ the language tag

Reply:
lol. fixed! https://github.com/docker/docs/pull/18860